### PR TITLE
Re-enable dependencies acceptance tests

### PR DESCRIPTION
**Pull request by danyf90**
Resolves https://github.com/tuist/tuist/issues/YYY

### Short description 📝

> Describe here the purpose of your PR.

### How to test the changes locally 🧐

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

### Contributor checklist ✅

- [ ] The code has been linted using run `make workspace/lint-fix`
- [ ] The change is tested via unit testing or acceptance testing, or both
- [ ] The title of the PR is formulated in a way that is usable as a changelog entry
- [ ] In case the PR introduces changes that affect users, the documentation has been updated

### Reviewer checklist ✅

- [ ] The code architecture and patterns are consistent with the rest of the codebase
- [ ] Reviewer has checked that, if needed, the PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, and the title is usable as a changelog entry

### CODE REVIEW

File: fixtures/app_with_spm_dependencies/App/Project.swift [hunk 1]
```diff
@@ -30,19 +30,6 @@ let project = Project(
                 .sdk(name: "c++", type: .library, status: .required),
                 .external(name: "Alamofire"),
                 .external(name: "ComposableArchitecture"),
-                .external(name: "FacebookCore"),
-                .external(name: "FirebaseAnalytics"),
```
Comment (fortmarek) [inline fixtures/app_with_spm_dependencies/App/Project.swift:34]: Should we consider keeping Firebase in given this dependency is a popular one that we broke in the past? Ofc depending on how much CI time it adds.

**Review by fortmarek — APPROVED** [review]

**Review by pepicrft — APPROVED** [review]

**Comment by pepicrft** [comment]
@danyf90 do you need a hand with this one?

**Comment by danyf90** [comment]
> @danyf90 do you need a hand with this one?

Yep, feel free to take over if you have time 🙏

**Comment by fortmarek** [comment]
Superseded by: https://github.com/tuist/tuist/pull/5802

**Pull request closed without merging**